client/src/components/geneExpression/gene.tsx

```diff
@@ -91,41 +91,27 @@ class Gene extends React.Component<Props, State> {
 
   handleDisplayGeneInfo = async (): Promise<void> => {
     // @ts-expect-error ts-migrate(2339) FIXME: Property 'dispatch' does not exist on type 'Readon... Remove this comment to see the full error message
-    const { dispatch, gene, geneId, isGeneInfo } = this.props;
+    const { dispatch, gene, geneId } = this.props;
     track(EVENTS.EXPLORER_GENE_INFO_BUTTON_CLICKED, {
       gene,
     });
 
-    if (isGeneInfo) {
-      dispatch({
-        type: "clear gene info",
-      });
-      return;
-    }
-
     dispatch({
       type: "load gene info",
       gene,
     });
 
+    dispatch({ type: "toggle active info panel", activeTab: "Gene" });
+
     const info = await actions.fetchGeneInfo(geneId, gene);
     if (!info) {
-      dispatch({
-        type: "open gene info",
-        gene,
-        url: "",
-        name: "",
-        synonyms: [],
-        summary: "",
-        infoError: "fetch gene info failed",
```

Comment:
maybe I'm missing something, but it seems like by removing this dispatch, we no longer tell user a gene info fetch as failed?

Reply:
this was kind of a workaround to trigger empty state in the UI - no longer needed with the new revised logic introduced in `geneInfo/index.tsx`

"open gene info" will still return `infoError` if it occurs and will affect the UI representation accordingly.